dts/bindings/clock/st,stm32-clock-mux.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2022, Linaro ltd
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+        STM32 Clock multiplexer
+        Describes a clock multiplexer, such as per_ck on STM32H7 or
+        CLK48 on STM32L5.
+        The only property of this node is to select a clock input.
+        For instance:
+                &perck {
+                        clocks = <&rcc STM32_SRC_HSI_KER CKPER_SEL(0)>;
+                        status = "okay";
+                };
+
+compatible: "st,stm32-clock-mux"
+
+properties:
```

Comment:
Most properties are available within the `base.yaml`.
Shouldn't the bindings be based on it?

Reply:
I intentionally didn't use base.yaml in order to limit the number of properties, as this kind of device has only one  purpose which is to set a clk source.

Reply:
Makes sense. 

I don't know if there is any policy regarding bindings and using shared inclusions. Shared properties need to be kept in sync. That increases the error possibility in the future.

Reply:
This is right. Fixed